docs/content/latest/explore/security/security.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Security
+headerTitle: Security
+linkTitle: Security
+description: Overview of security in YugabyteDB.
+headcontent: Security in YugabyteDB.
+image: /images/section_icons/index/secure.png
+aliases:
+  - /latest/explore/security/
+menu:
+  latest:
+    identifier: explore-security
+    parent: explore
+    weight: 930
+isTocNested: true
+showAsideToc: true
+---
+
+Like PostgreSQL, YugabyteDB provides security in multiple ways:
+
+* **Authentication** - limit access to the database to clients with proper credentials
```

Comment:
I'm not sure if it's "client" as opposed to "server", or "client" as a "user"

Reply:
Reading the Postgresql docs, 'client' appears to refer to whatever is connecting to the database, and a 'user' is defined in the db and given roles etc. A client connects using a database user name. So it is 'client authentication' and 'user authorization'